components/KeyManagerPermissions/KeyManagerPermissions.tsx

```diff
@@ -39,20 +39,57 @@ const KeyManagerPermissions: React.FC = () => {
       <h2 className="title is-2">Permissions</h2>
       <article className="message is-info">
         <div className="message-body">
-          This tool will encode/decode permissions according to{' '}
-          <a href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager">
-            LSP6 KeyManager Standard
-          </a>{' '}
-          smart contract and attempt to match them with their{' '}
-          <a href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md">
+          This tool will encode and decode
+          <a
+            href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#address-permissions"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            permissions
+          </a>
+          based on the
+          <a
+            href="https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            LSP6 KeyManager
+          </a>
+          standardization to match them with their
+          <a
+            href="https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md"
+            target="_blank"
+            rel="noreferrer"
+            className="ml-1"
+          >
             LSP2 ERC725YJSONSchema
           </a>
-          .<br />
-          The erc725.js lib provides a{' '}
-          <a href="https://docs.lukso.tech/tools/erc725js/classes/ERC725#encodepermissions">
-            decodePermissions
-          </a>{' '}
-          method to decode the permissions.
+          .
+        </div>
+      </article>
+      <article className="message">
+        <div className="message-body">
+          It`s using the
+          <a
+            href="https://docs.lukso.tech/tools/erc725js/classes/ERC725#encodepermissions"
+            target="_blank"
+            rel="noreferrer"
+            className="mx-1"
+          >
+            encodePermissions
+          </a>
+          function of the
+          <a
+            href="https://www.npmjs.com/package/@erc725/erc725.js"
```

Comment:
Can you use the getting started page of erc725js page on our docs instead? (Cf other comment in other PR)

Reply:
cf other pr - ok